packages/google_sign_in/google_sign_in/lib/google_sign_in.dart

```diff
@@ -146,7 +139,7 @@ class GoogleSignIn {
   /// Initializes global sign-in configuration settings.
   ///
   /// The [signInOption] determines the user experience. [SigninOption.games]
-  /// must not be used on iOS.
+  /// must not be used on iOS or web.
```

Comment:
Maybe we should just say:
`is only supported on Android.`

Reply:
Good call, much clearer.